dockerfiles/Dockerfile.apisix-openresty.rpm

```diff
@@ -3,19 +3,21 @@ ARG IMAGE_TAG="7"
 
 FROM ${IMAGE_BASE}:${IMAGE_TAG}
 
-COPY build-apisix-openresty-centos7.sh /tmp/build-apisix-openresty-centos7.sh
+COPY ./utils/build-common.sh /tmp/build-common.sh
 COPY build-apisix-openresty.sh /tmp/build-apisix-openresty.sh
 COPY ./utils/determine-dist.sh /tmp/determine-dist.sh
 
 WORKDIR /tmp
 
 ARG version
+ARG debug
```

Comment:
For arguments defined in dockerfile, it's common to use in upper characters.

Reply:
Much appreciated your kind reminder. XD